iroh/src/magicsock/relay_actor.rs

```diff
@@ -213,7 +209,7 @@ impl ActiveRelayActor {
                         relay_client.send(msg.node_id, msg.packet).await
                     };
                     relay_send_fut.as_mut().set_future(fut);
-                    self.last_write = Instant::now();
+                    inactive_timeout.as_mut().reset(Instant::now() + RELAY_INACTIVE_CLEANUP_TIME);
```

Comment:
not sure, but you could use `Interval` and call `reset()` here

Reply:
Yeah ok, it's a close call because interval is a little weird here because you only ever use the first tick.  But the code does look simpler when using interval so let's go with it.